collections/src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -266,6 +284,19 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
     */
   def to[C1](factory: Factory[A, C1]): C1 = factory.fromSpecific(toIterable)
 
+  def toList: immutable.List[A] = immutable.List.from(toIterable)
+
+  def toVector: immutable.Vector[A] = immutable.Vector.from(toIterable)
+
+  def toMap[K, V](implicit ev: A <:< (K, V)): immutable.Map[K, V] =
+    immutable.Map.from(toIterable.asInstanceOf[Iterable[(K, V)]])
+
+  def toSet[B >: A]: immutable.Set[B] = immutable.Set.from(toIterable)
+
+  def toSeq: immutable.Seq[A] = immutable.Seq.from(toIterable)
+
+  def toIndexedSeq[B >: A]: immutable.IndexedSeq[B] = immutable.IndexedSeq.from(toIterable)
```

Comment:
In this one too, we should remove the `B` type parameter.

Also, I’d be in favor of deprecating `toIndexedSeq`, `toList` and `toVector`, but that’s another issue.